controllers/eventbus/validate.go

```diff
@@ -36,5 +36,14 @@ func ValidateEventBus(eb *v1alpha1.EventBus) error {
 			return fmt.Errorf("invalid spec: a jetstream eventbus requires at least 3 replicas")
 		}
 	}
+	if x := eb.Spec.Kafka; x != nil {
+		if x.Exotic == nil {
+			return fmt.Errorf("\"exotic\" must be defined")
+		}
+		e := x.Exotic
+		if e.URL == "" {
```

Comment:
```suggestion
		if x.Exotic.URL == "" {
```

One less variable to define